.github/workflows/bb-run-tests.yml

```diff
@@ -6,10 +6,36 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
-      # First ensure we've got babashka installed
+
+      - name: Install Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Install Clojure
```

Comment:
Since we've already got babashka installed, there's no need to do this. See comment below.